zerver/webhooks/front/doc.md

```diff
@@ -2,36 +2,23 @@ Front lets you manage all of your communication channels in one place,
 and helps your team collaborate around every message. Follow these steps
 to recieve Front notifications without leaving Zulip!
 
-{!create-stream.md!}
+1. {!create-stream.md!}
 
-{!create-bot-construct-url.md!}
+1. {!create-bot-construct-url-indented.md!}
 
-Then, log into your Front account, and do the following:
+1. Go to the **Settings** page of your Front account. Click on the
+**Integrations** tab, and enable the **Webhooks** integration. Click on
+the **Rules** tab, and add a new rule.
 
-1. Go to the **Settings** of your organization.
-   ![](/static/images/integrations/front/001.png)
+1. Select the events you would like to be notified about. Set the URL of
+the target webhook to the URL constructed above.
 
-2. Enable webhook notifications.
-   ![](/static/images/integrations/front/002.png)
-   ![](/static/images/integrations/front/003.png)
-
-3. Pick the types of events for which you want to receive notifications,
-and set the URL you construsted for the Front bot as a target webhook.
-   ![](/static/images/integrations/front/004.png)
-   ![](/static/images/integrations/front/005.png)
-
-Finally, go to the **Settings** of your Zulip organization, and add a new
-filter on the **Filter settings** page.
-
-**Pattern**: `cnv_(?P<id>[0-9a-z]+)`.
-**URL format string**: `https://app.frontapp.com/open/cnv_%(id)s`.
-
-![](/static/images/integrations/front/006.png)
-
-This step is necessary to set a link to a Front conversation as a name of
-the corresponding topic in Zulip.
+1. Open the **Settings** of your Zulip organization, and add a new filter
+on the **Filter settings** page. This step is necessary to map Front
+conversations to topics in Zulip.
+* Pattern: `cnv_(?P<id>[0-9a-z]+)`.
+* URL format string: `https://app.frontapp.com/open/cnv_%(id)s`.
```

Comment:
The `Pattern` and `URL format string` bullet points are being rendered as independent numbered steps. I would probably just make it a part of the same paragraph in the last point, like so:
```
Set **Pattern** to `cnv_(?P<blah blah)`. Set the URL format string to `<url goes here>`. This step is necessary to map Front conversations to topics in Zulip. 
```

Reply:
Just for some background, nested lists can be confusing and Markdown doesn't really interact with Bootstrap too sensibly when rendering nested lists (for God knows what reason). So let's just stick to one level! :)

Reply:
Done! I hope you don't mind, I also combined these two steps:
```
1. Go to the **Settings** page of your Front account. Click on the
**Integrations** tab, and enable the **Webhooks** integration.

1. Click on the **Rules** tab, and add a new rule. Select the events you
would like to be notified about. Set the URL of the target webhook to the
URL constructed above.
```
into one:
```
1. Go to the **Settings** page of your Front organization. Click on the
**Integrations** tab, and enable the **Webhooks** integration. Click on
the **Rules** tab, and add a new rule. Select the events you would like to
be notified about. Set the URL of the target webhook to the URL
constructed above.
```
This way, we have two major steps: one explains the setup process on the Front's side, another—on the Zulip's side. It seems more intuitive to me, but I'll revert it if necessary.